### PR TITLE
Carthage

### PR DESCRIPTION
**Pull request by adly-holler**
I was working on a toy iOS project to get more practice with Few.swift, and I had trouble getting it imported into so ta-da!

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -0,0 +1 @@
+github "joshaber/SwiftBox" "c11e489"
```
Comment (adly-holler) [inline Cartfile]: This is currently pointing to the tip of master. We were previously pointing to the prior commit, but the latest one seems innocuous. Might be time for a SwiftBox release just so we can point to it.

Comment (joshaber) [inline Cartfile]: Ah, yeah, SwiftBox needs a few other changes so I can take care of that later :+1:

**Comment by joshaber** [comment]
:sunglasses:

**Comment by adly-holler** [comment]
Woohoo!

**Pull request merged**